client/styles/components/_forms.scss

```diff
@@ -52,7 +52,6 @@
 
 .form__input {
   width: 100%;
-  min-width: #{360 / $base-font-size}rem;
```

Comment:
I gave you bad instructions here because I didn't realize that deleting this line will cause the inputs to be too narrow on large screens. 🤦‍♀️ 

Thinking this through...
Right now it is 360px wide always.  This is a problem on small screens where 360px is too big.  We want it to be 360px when possible but not too large to fit.  The `width: 100%` alone doesn't do this because there is no width on the parent container (it will be too small on large screens).  Doing a fixed width with `max-width: 100%` also doesn't work unless we also put a `max-width` on the container (it will overflow on small screens).

Option A - We put it back as it was before and merge this PR which fixes the background color issue.

Option B - The not perfect but very good fix.
```
.form__input {
  width: #{360 / $base-font-size}rem;
  max-width: 90vw;
```

The "perfect" fix would involve a `max-width: 100%;`, but that involves changing the CSS on a few ancestors and it becomes beyond the scope of this PR.

Reply:
Hey, thank you for the review. Right now, I will work on option B.

> The "perfect" fix would involve a max-width: 100%;, but that involves changing the CSS on a few ancestors and it becomes beyond the scope of this PR.

Sure, I will work on the perfect fix, but not immediately. I will address it later.

Reply:
Re: "perfect" fix, I actually have a CSS which I think is good but this didn't seem like the right PR for it.
```
.form__input {
  width: #{360 / $base-font-size}rem;
  max-width: 100%;
```
```
.form-container {
  max-width: 100vw;
  padding: #{20 / $base-font-size}rem;
```
```
.form-container__content {
  max-width: 100%;
  & > * {
    max-width: 100%;
  }
```